timemachine/cpp/src/nonbonded_all_pairs.cu

```diff
@@ -439,7 +439,7 @@ void NonbondedAllPairs<RealType, Interpolated>::execute_device(
 
 template <typename RealType, bool Interpolated>
 void NonbondedAllPairs<RealType, Interpolated>::du_dp_fixed_to_float(
-    const int N, const int P, const unsigned long long *du_dp, double *du_dp_float) {
+    const int N, const int, const unsigned long long *du_dp, double *du_dp_float) {
```

Comment:
not sure why some of these are removed...

Reply:
Addressed by https://github.com/proteneer/timemachine/pull/697#discussion_r845461217